src/components/info-tile/info-tile.stories.js

```diff
@@ -0,0 +1,82 @@
+import { html } from 'lit-html';
+import '../info-tile';
+import { docsIntro, CATEGORIES, TYPES } from '../../../.storybook/utils.js';
+
+export default {
+  title: 'custom-elements/vl-info-tile',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ['info-tile'],
```

Comment:
Er staat nog steeds vermeld dat er een stylesheet geïmporteerd moet worden bij gebruik, wat niet waar is.